presenters/adlink.go

```diff
@@ -0,0 +1,25 @@
+package presenters
+
+import (
+	"go-google-scraper-challenge/models"
+)
+
+type AdLinkCollectionPresenter struct {
+	Position        string
+	Links           []string
+	LinkCount       int
+}
+
+func GetAdLinkCollection(adlink []*models.AdLink) map[string][]string {
+	adLinkCollection := map[string][]string{
+		models.AdLinkPositionTop: []string{},
```

Comment:
Base on our convention, it prefer `var s []string` to declare empty slice. What do you think?

![Screen Shot 2021-06-07 at 12 44 11 PM](https://user-images.githubusercontent.com/6076762/120964804-12d27d00-c78e-11eb-9f41-8e4296d0bc02.png)

Reply:
fixed in [7f4867e](https://github.com/carryall/go-google-scraper-challenge/pull/73/commits/7f4867e2deebd90680f1f00e8c5b60a89a6484b0)

Reply:
@carryall I accualy meant `[]string{}` of `models.AdLinkPositionTop: []string{},` not the parent one( `map[string][]string{`) which is not empty slice.

Reply:
Ahh I see 😄  updated in [c71cb52](https://github.com/carryall/go-google-scraper-challenge/pull/73/commits/c71cb52700d554b3a837a53e4365f7d9e538cb81)